app/Providers/AppServiceProvider.php

```diff
@@ -17,8 +19,11 @@ class AppServiceProvider extends ServiceProvider
     public function register()
     {
         $this->app->bind(RequestActivityLoggerInterface::class, function () {
-            return $this->app->make(DummyRequestActivityLogger::class);
-            // Environment type. Return sau Debug sau Production
+            if(strtolower(env('APP_ENV')) === 'production')
```

Comment:
Can't accept, the env helper is not available in cache environment, so it is broken.

Reply:
Now using config() helper instead of env().